### ISSUE TITLE
Refactor API URL templates

### ISSUE THREAD

**Issue by sirex** [atviriduomenys/spinta#123]
Current API URL formatting has some issue, the main issue, that it is difficult to recognize ID, I have to run quite complicated detection algorithm in order, to determine if a URL path component is an ID. Also model naming convention is that model names must be in singular form, but then in URL this is not reflected.

To fix these issue, a new URL form could look like this:

| old                   | new                        | Meaning                                                    |
| --------------------- | -------------------------- | ---------------------------------------------------------- |
| /ns/Model/:schema     | /ns/Model                  | Show model schema.                                         |
| /ns/Model             | /ns/Model/:all             | Get all model objects.                                     |
| /ns/Model/UUID        | /ns/Model/:one/UUID        | Get single model object by UUID.                           |
| /ns/Model.prop        | /ns/Model/:all/prop        | Get all values of a model property of all objects.         |
| /ns/Model/UUID/prop   | /ns/Model/:one/UUID/prop   | Get a single model property of a single object.            |
| -                     | /ns/Model/:one?query       | Get a single model object by a given query.                |
| -                     | /ns/Model/:one/prop?query  | Get a single model property by a given query.              |
| /ns/Model/:format/ext | /ns/Model/:all.ext         | Get all model objects in a given ext format.               |
| /ns/Model.nested.prop | /ns/Model/:all/nested/prop | Get all values of a nested model property of all objects.  |

Also with this, scope naming would also have to changes:

| old                    | new                          | Meaning                           |
| ---------------------- | ---------------------------- | --------------------------------- |
| -                      | spinta:ns/Model/:schema/:put | Update model schema.              |
| spinta_ns_model_getall | spinta:ns/Model/:all/:get    | Get all model objects.            |
| spinta_ns_model_getone | spinta:ns/Model/:one/:get    | Get one model object by UUID.     |
| spinta_getall          | spinta::all/:get             | Get all objects of all models.    |

Basically scopes would become exactly the same as URLs, only with `spinta:` prefix and `:get`, `:post`, `:put`, `:patch` or `:delete` HTTP methods.
